hardware_interface/include/hardware_interface/handle.hpp

```diff
@@ -15,42 +15,51 @@
 #ifndef HARDWARE_INTERFACE__HANDLE_HPP_
 #define HARDWARE_INTERFACE__HANDLE_HPP_
 
+#include <limits>
+#include <memory>
 #include <string>
+#include <utility>
+#include <variant>
 
+#include "hardware_interface/hardware_info.hpp"
 #include "hardware_interface/macros.hpp"
+#include "hardware_interface/visibility_control.h"
 
 namespace hardware_interface
 {
+
+typedef std::variant<double> HANDLE_DATATYPE;
```

Comment:
Let's use an alias declaration?

```suggestion
using HANDLE_DATATYPE = std::variant<double>;
```

Reply:
yes right, dont know why i used typedef in the first place